### ISSUE TITLE
RTR - communication

### ISSUE THREAD

**Issue by CDowland** [ufbmi/OMRSE#129]
Additional axioms requested. Please see the attached spreadsheet. 
[RTR - communication.xlsx](https://github.com/ufbmi/OMRSE/files/5031178/RTR.-.communication.xlsx)

**Comment by mbrochhausen** [comment]
In the dev version of d-acts I clarified the notion of social act and that Reinach does not intend being perceived to be a criterion for a social act. Bill found the quote that clearly indicates that. Hence, I do belief it would be ok to classify communication as a social act. Take a look at the improved definition of social act, which comes with a new comment containing the Reinach quote.

**Comment by CDowland** [comment]
When we were developing the term 'communication,' we decided to not limit it to conscious beings. Instead, we defined it in a way that is broad enough to include, for example, communication performed by or directed toward a computer that is not conscious. But social acts are performed by either conscious beings or organizations, and are directed toward either conscious beings or organizations. So, while many communication processes are indeed social acts, not all are (at least within the approach we decided to take). If we want 'communication' to be a subclass of 'social act,' then we'd need to define it differently, and would not be able to use it to represent communication between a person and non-conscious computer, or between two non-conscious computers.
